test/FileService.Validation.UnitTests/ImageRequestValidatorTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using FluentValidation.TestHelper;
+using LT.DigitalOffice.FileService.Models.Dto.Requests;
+using LT.DigitalOffice.FileService.Validation.Interfaces;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.FileService.Validation.UnitTests
+{
+    public class AddNewImageRequestValidatorTests
```

Comment:
Class name and file name have a difference

Reply:
Fixed